### ISSUE TITLE
Failed execution does not carry detailed information

### ISSUE THREAD

**Issue by lppedd** [usefulness/ktlint-gradle-plugin#176]
Thanks for the plugin! I needed a simple wrapper without too many abstractions, and this works nicely.  
There is only thing tho that could be improved: when a format task fails, I get messages like:
```
Execution failed for task ':zowe-imperative:formatKotlinJsMain'.
> There were multiple failures while executing work items
   > A failure occurred while executing io.github.usefulness.tasks.workers.KtlintWorker
      > 9:9 Expecting property name or receiver type
   > A failure occurred while executing io.github.usefulness.tasks.workers.KtlintWorker
      > 102:1 Expecting a top level declaration
```

You can see there is no info on the file where it happened.

**Comment by mateuszkwiecinski** [comment]
Interesting 👀 The plugin is just a wrapper around ktlint and AFAIR all the errors come from ktlint itself. From what understand this type of errors appear when the code doesn't compile, so, as a workaround, running regular compilation tasks should provide precise information about the source of the error.

I should be able to print at least the file path though 👀 I'll see what can be improved there 👍

**Comment by lppedd** [comment]
Yup the code did not compile. But if you could print the path where it fails, I'd be happy with that single enhancement!

Thanks!

**Comment by mateuszkwiecinski** [comment]
The fix should be available in [0.8.2](https://github.com/usefulness/ktlint-gradle-plugin/releases/tag/v0.8.2):

<img width="1360" alt="image" src="https://github.com/usefulness/ktlint-gradle-plugin/assets/36954793/c1184448-7ee0-4e05-9a30-1f5e7dd1cb91">

**Comment by lppedd** [comment]
Thank you!
